modules/apps/document-library/document-library-web/src/main/java/com/liferay/document/library/web/internal/display/context/FolderActionDisplayContext.java

```diff
@@ -552,13 +554,29 @@ private String _getModelResourceDescription() throws PortalException {
 		return themeDisplay.getScopeGroupName();
 	}
 
-	private String _getMoveFolderURL() {
+	private String _getMoveFolderURL() throws PortalException {
 		LiferayPortletResponse liferayPortletResponse =
 			_dlRequestHelper.getLiferayPortletResponse();
 
+		FolderItemSelectorURLProvider folderItemSelectorURLProvider =
+			new FolderItemSelectorURLProvider(
+				_httpServletRequest,
+				(ItemSelector)_httpServletRequest.getAttribute(
```

Comment:
this is is causing because itemselector is not found
```
 ERROR [http-nio-8080-exec-8][IncludeTag:119] Current URL /group/guest/~/control_panel/manage?p_p_id=com_liferay_document_library_web_portlet_DLAdminPortlet&p_p_lifecycle=2&p_p_state=maximized&p_p_mode=view&p_p_resource_id=%2Fdocument_library%2Finfo_panel&p_p_cacheability=cacheLevelPage&_com_liferay_document_library_web_portlet_DLAdminPortlet_folderId=0&_com_liferay_document_library_web_portlet_DLAdminPortlet_repositoryId=20117&p_p_auth=ORQaJUgv generates exception: java.lang.RuntimeException: java.lang.RuntimeException: java.lang.RuntimeException: java.lang.NullPointerException
java.lang.NullPointerException: null
	at com.liferay.document.library.web.internal.util.FolderItemSelectorURLProvider._getItemSelectorURL(FolderItemSelectorURLProvider.java:119) ~[bundleFile:?]
	at com.liferay.document.library.web.internal.util.FolderItemSelectorURLProvider.getSelectMoveToFolderURL(FolderItemSelectorURLProvider.java:77) ~[bundleFile:?]
	at com.liferay.document.library.web.internal.display.context.FolderActionDisplayContext._getMoveFolderURL(FolderActionDisplayContext.java:571) ~[bundleFile:?]
	at com.liferay.document.library.web.internal.display.context.FolderActionDisplayContext.lambda$null$3(FolderActionDisplayContext.java:109) ~[bundleFile:?]
	at com.liferay.frontend.taglib.clay.servlet.taglib.util.DropdownItemList.add(DropdownItemList.java:61) ~[bundleFile:?]
	at com.liferay.frontend.taglib.clay.servlet.taglib.util.DropdownItemListBuilder$DropdownItemListWrapper.add(DropdownItemListBuilder.java:199) ~[bundleFile:?]
	at com.liferay.frontend.taglib.clay.servlet.taglib.util.DropdownItemListBuilder.add(DropdownItemListBuilder.java:51) ~[bundleFile:?]
	at com.liferay.document.library.web.internal.display.context.FolderActionDisplayContext.lambda$getActionDropdownItems$14(FolderActionDisplayContext.java:106) ~[bundleFile:?]
	at com.liferay.frontend.taglib.clay.servlet.taglib.util.DropdownItemList.addGroup(DropdownItemList.java:106) ~[bundleFile:?]
	at com.liferay.frontend.taglib.clay.servlet.taglib.util.DropdownItemListBuilder$DropdownItemListWrapper.addGroup(DropdownItemListBuilder.java:272) ~[bundleFile:?]
	at com.liferay.document.library.web.internal.display.context.FolderActionDisplayContext.getActionDropdownItems(FolderActionDisplayContext.java:103) ~[bundleFile:?]
	at org.apache.jsp.document_005flibrary.folder_005faction_jsp._jspService(folder_005faction_jsp.java:452) ~[?:?]
	at org.apache.jasper.runtime.HttpJspBase.service(HttpJspBase.java:111) ~[jasper.jar:9.0.83]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:623) ~[servlet-api.jar:4.0.FR]
	at org.apache.jasper.servlet.JspServletWrapper.service(JspServletWrapper.java:411) ~[jasper.jar:9.0.83]
	at org.apache.jasper.servlet.JspServlet.serviceJspFile(JspServlet.java:473) ~[jasper.jar:9.0.83]
	at org.apache.jasper.servlet.JspServlet.service(JspServlet.java:377) ~[jasper.jar:9.0.83]
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:623) ~[servlet-api.jar:4.0.FR]
	at com.liferay.portal.osgi.web.servlet.jsp.compiler.internal.JspServlet.service(JspServlet.java:310) ~[bundleFile:?]
	at com.liferay.portal.osgi.web.servlet.jsp.compiler.internal.JspServlet.service(JspServlet.java:319) ~[bundleFile:?]
```

Reply:
solved on https://github.com/liferay-lima/liferay-portal/pull/5030/commits/dc2682e4a7b46c8becb701149177563fb22b6f92

Please review